jshint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# 1st sed: Replace top lines including first beginning with '}' with empty string
+# 2nd sed: Insert newline before every '/** BEGIN TEST'
+# 3rd sed: Delete from first line containing 'BEGIN TEST' to end of file
+# Run the npm-installed jshint
+# No errors -> print 'filename.js OK'
+# Errors -> Set $err to non-zero value
+for f in *.js; do
+      sed '1,/^}/ s/.*//' $f \
+    | sed 's,/\*\* BEGIN TEST,\n\0,' \
+    | sed '/BEGIN TEST/,$ d' \
+    | tee foo.js \
```

Comment:
Ist sie nicht, das war nur zum testen, hab ich übersehen gestern :zzz:. `tee` leitet STDIN in eine Datei und nach STDOUT.